testrunner.Makefile

```diff
@@ -0,0 +1,20 @@
+ifndef __TESTRUNNER_MAKEFILE__
+
+__TESTRUNNER_MAKEFILE__ := included
+
+
+makefile_dir := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+include $(makefile_dir)/common.Makefile
+
+
+.build/testrunner: | .build
+	$(call print_target)
+	cd $(MARKETPLACE_TOOLS_PATH)/testrunner \
+	  && bazel build //runner:main \
+	  && rm -rf tmp \
```

Comment:
I think we should put temp in the `.build` tree just to not pollute the source tree. We can use `$@/tmp` instead.

Reply:
We cannot use `$@/tmp` for a `tmp` dir, because we use `touch $@` as a mark file for a target, so `$@` is a file, not a directory.

Also, `tmp` is in `testrunner/.gitignore`, and those steps are just steps from `testrunner/README.md`.

How do you feel about leaving this as is? And how about using `.build/tmp`, maybe this is a best solution in this case?

Reply:
Let's use `.build/tmp/testrunner`.

Ideally building shouldn't modify source, even if the files are in gitignore, because we'll want to remove all build artifacts with `make clean` too. So I do want to try to find a way not to pollute source.

Reply:
Done.